packages/calcite-components/src/components/dropdown/resources.ts

```diff
@@ -4,4 +4,6 @@ export const SLOTS = {
 
 export const CSS = {
   calciteDropdownContent: "calcite-dropdown-content",
+  calciteDropdownGroup: "calcite-dropdown-group",
```

Comment:
These classes don't appear to belong here as they are not used in the component's rendering.